assets/scss/styles.css

```diff
@@ -0,0 +1 @@
+:root{--clr-primary: #000;--clr-secondary: #000;--clr-teritary: #fff;--clr-grey: #ececec;--clr-white: #fff;--clr-black: #000;--fs-xl: clamp(2rem, 1.5652rem + 2.3188vw, 4rem);--fs-l: clamp(1.5rem, 1.1739rem + 1.7391vw, 3rem);--fs-m: 1.5rem;--fs-r: 1rem;--fs-s: 0.85rem;--fs-xs: 0.5rem;--spacing-xxl: 4rem;--spacing-xl: 3rem;--spacing-l: 2rem;--spacing-m: 1.5rem;--spacing-r: 1rem;--spacing-s: 0.75rem;--spacing-xs: 0.5rem;--spacing-xxs: 0.25rem;--animated-cubicBez: 0.4s cubic-bezier(0.68, -0.6, 0.32, 1.6);--animated-150ms: 150ms}@media(prefers-reduced-motion){*{transition:none !important}}*,*::before,*::after{box-sizing:border-box}html,body{height:100%;margin:0;padding:0}body{line-height:1.5;-webkit-font-smoothing:antialiased}img,picture,video,canvas,svg{display:block;max-width:100%}img{width:100%}input,button,textarea,select{font:inherit}button{-webkit-appearance:none;appearance:none;background-color:rgba(0,0,0,0);border:none;padding:0}p,h1,h2,h3,h4,h5,h6{overflow-wrap:break-word}#root,#__next{isolation:isolate}ul{list-style:none;padding:0;margin:0}/*# sourceMappingURL=styles.css.map */
```

Comment:
Am I right by thinking this file will get automatically generated using the CLI via `make` when you make a change to any of the `scss` files?

Reply:
Yes, this is the generated minified CSS.